specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/examples/StorageAccountCreateOrUpdateUserAssignedEncryptionIdentityWithCMK.json

```diff
@@ -0,0 +1,122 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "res9101",
+    "accountName": "sto4445",
+    "api-version": "2020-08-01-preview",
+    "parameters": {
+	   "identity": {
+		 "type": "UserAssigned", 
+		 "userAssignedIdentities": { 
+			 "/subscriptions/{subscription-id}/resourceGroups/res9101/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{managed-identity-name}": {
+		   } 
+	   }
+      },
+      "sku": {
+        "name": "Standard_LRS"
+      },
+      "kind": "Storage",
+      "location": "eastus",
```

Comment:
Please only input the required parameter for update user identity in update account sample.
This is one reason make the [swagger validation](https://github.com/Azure/azure-rest-api-specs/pull/11602/checks?check_run_id=1795872136) failed for this PR, like location is not allowed in update account.

Reply:
Thanks WeiWei. I removed the "location" property from the update example.